app/api/server/v1/chat.js

```diff
@@ -583,10 +598,13 @@ API.v1.addRoute('chat.syncThreadMessages', { authRequired: true }, {
 		if (!canAccessRoom(room, user)) {
 			throw new Meteor.Error('error-not-allowed', 'Not Allowed');
 		}
+
+		const threadQuery = Object.assign({}, query, { tmid }, { _updatedAt: { $gt: updatedSinceDate } });
```

Comment:
This `{ _updatedAt: { $gt: updatedSinceDate } }` will not affect the filter for the deleted messages?